lib/components/Portal/Portal.tsx

```diff
@@ -0,0 +1,20 @@
+import { invariant } from '@autoguru/utilities';
+import { FunctionComponent, ReactNode } from 'react';
+import { createPortal } from 'react-dom';
+
+import { usePortalContext } from './PortalProvider';
+
+interface Props {
+	children: ReactNode;
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e32d4f0292c57f63310af65a7fb3708155cf8083/types/react/index.d.ts#L517-L523

remove this children thing